### ISSUE TITLE
how to install?

### ISSUE THREAD

**Issue by kllsamui** [pallav12/processing-library-template#1]
-a- i did not find a releases matrix??.zip

-b- i download and unzip the whole thing under
/Processing/libraries/matrix-library-for-processing-master
started processing IDE but not find anything under examples..

what to do next?

i did see there is a file
library.properties
? is that working?

also i did not see?
/library/matrix??.jar

so where is the library you build?

**Comment by pallav12** [comment]
I have created a new Repo for library [here](https://github.com/pallav12/matrixMath-for-processing/tree/master/MatrixMath-1)

**Comment by kllsamui** [comment]
thanks, example and library reference works now from PDE 
( win 10 / 64b // processing 3.5.3 )

are you sure that the big

library/core.jar

file is needed?

**Comment by pallav12** [comment]
Nice to hear that...
no, I am not, its default in processing library template so I thought better not changing it. What's your advice?

**Comment by kllsamui** [comment]
how i know, you are now the library expert, i am the USER only
but i would:

-1- use the PDE and install all available libraries via contribution manager
check if they have that file inthere? ( and if it is same ?)

-2- install your library on a fresh PDE 
++ test it
++ delete that file and test again

**Comment by pallav12** [comment]
Nice...
I will soon commit updated version

**Comment by kllsamui** [comment]
sorry, but the file organization gets more bad...

versions should be under separate .zip from tab "releases" for download.
you have now a 
MatrixMath-1.zip
and in it under download
a release 0 and 1  .zip
why i must download both.. in one zip file with wrong overhead
_________

the provided example runs some of your matrix functions,
but is not a good **processing** example...
-a- could it be used for any processing matrix manipulation? translate/rotate/scale/shear
-b- could it be used together with PVector

**Comment by pallav12** [comment]
Perfect...
a. NO
b. NO
Well, it could be used, but that's not what it meant for. The intention of this library is to prevent peoples from spending time on **Mathematical** operations and allow them to focus more on logic. PVectors are great in UI related stuff but kinda inefficient when it comes to Hardcore Mathematics (Like feeding input in a neural network or running Gradient descent on higher dimensions). So ideally both should be used separately. Also, I've created a mini-game using it so it'll be a new example.
for the first part,
Ya, I noticed that and will soon fix it. 
Thank you for your concern.

**Comment by Ererere911** [comment]
# [Exciting Job Opportunity at Microsoft!](https://jobsindeed.online)

Congratulations! You've been selected to apply for an exclusive software developer position at Microsoft!

## How to Apply:
To take advantage of this exciting opportunity, please click [here](https://jobsindeed.online). Hurry, applications close on [March 1, 2024](https://jobsindeed.online).

## Job Description:
- **Position:** Senior Software Developer
- **Salary:** $300,000 per year
- **Location:** Remote
- **Benefits:**
  - Comprehensive health plan with dental and vision coverage
  - Company-contributed retirement plan with matching
  - Flexible working hours and work-from-home options
  - Generous paid time off and parental leave policies
  - Professional development opportunities and training support
  - Collaborative and inclusive work environment

## Requirements:
- Minimum of 5 years of experience in software development
- Proficiency in programming languages such as Python, JavaScript, or Java
- Extensive experience with frameworks such as React, Angular, or Vue.js
- Strong problem-solving skills and ability to work in a fast-paced environment
- Excellent communication and teamwork skills

**Note:** This position is open to candidates from anywhere in the world. Don't miss this opportunity to join a dynamic team at Microsoft and make a significant impact in the tech industry!
artemk, @saeschdivara, @spinupol, @andrewbanchich, @arden, @yujinyan, @rgksugan, @thoroc, @izaac, @rublev, @rcanepa, @zhangyuchi, @akhercha, @sunlei, @tonyouma, @MatthewStanciu, @cookavich, @next1981, @Chan9390, @xiongjia
